pkg/kapp/cmd/core/deps_factory.go

```diff
@@ -138,12 +139,16 @@ func (f *DepsFactoryImpl) newWarningHandler() rest.WarningHandler {
 	}
 	options := rest.WarningWriterOptions{
 		Deduplicate: true,
-		Color:       f.WarningsColor,
+		Color:       f.WarningsColor && envVarsAllowColors(),
```

Comment:
Is it fine to use this environment variable here? 
I was thinking of disabling the colouring of warnings for now and later on implement a new method in ui library to display warnings, which will control the colouring as well. IMO this would be more neat and clean.